tests/helpers.py

```diff
@@ -65,6 +65,7 @@ def validate_conda_build_args(recipe, expect_recipe=None, expect_config=None, ex
         config = kwargs['config']
         for term, value in expect_config.items():
             assert hasattr(config, term)
+            print("{}: {}".format(term, getattr(config, term)))
```

Comment:
Do you still want this new print statement?

Reply:
Nope. Thanks.